src/Data/Version/Haskell.purs

```diff
@@ -9,24 +9,24 @@
 -- | though, you should probably be using `Data.Version`.
 module Data.Version.Haskell where
 
-import Prelude
+import Prelude (class Show, class Ord, class Eq, show, compare, flip, pure, bind, map, (<>), (==), (&&), (<<<), ($), (*>), (||), (<$>))
```

Comment:
Can we keep the Prelude import implicit here? IIRC you're allowed one implicit import before you start getting warnings.